packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -571,7 +571,7 @@ class SliverChildListDelegate extends SliverChildDelegate {
   /// [addSemanticIndexes], and [semanticIndexCallback] arguments must not be
   /// null.
   ///
-  /// If the order of children` never changes, consider using the constant
+  /// If the order of children never changes, consider using the constant
```

Comment:
Nit:

```suggestion
  /// If the order of children never changes, consider using the constant.
```

Reply:
I think the rest of the sentence is on the next line.

Reply:
Oh! The github UI cut it off. lol 🙃